src/utils/collections.zig

```diff
@@ -311,16 +407,16 @@ test "SortedSet range" {
 test find {
     const items: [4]u8 = .{ 1, 3, 3, 5 };
     inline for (.{ .any, .first, .last }) |w| {
-        try expect(find(u8, &items, 0, w) == .less);
-        try expect(find(u8, &items, 1, w).found == 0);
-        try expect(find(u8, &items, 2, w).after == 0);
-        try expect(find(u8, &items, 4, w).after == 2);
-        try expect(find(u8, &items, 5, w).found == 3);
-        try expect(find(u8, &items, 6, w) == .greater);
+        try expect(find(u8, &items, 0, w, std.math.order) == .less);
+        try expect(find(u8, &items, 1, w, std.math.order).found == 0);
+        try expect(find(u8, &items, 2, w, std.math.order).after == 0);
+        try expect(find(u8, &items, 4, w, std.math.order).after == 2);
+        try expect(find(u8, &items, 5, w, std.math.order).found == 3);
+        try expect(find(u8, &items, 6, w, std.math.order) == .greater);
     }
-    expect(find(u8, &items, 3, .any).found == 1) catch {
-        try expect(find(u8, &items, 3, .any).found == 2);
+    expect(find(u8, &items, 3, .any, std.math.order).found == 1) catch {
+        try expect(find(u8, &items, 3, .any, std.math.order).found == 2);
     };
-    try expect(find(u8, &items, 3, .first).found == 1);
-    try expect(find(u8, &items, 3, .last).found == 2);
+    try expect(find(u8, &items, 3, .first, std.math.order).found == 1);
+    try expect(find(u8, &items, 3, .last, std.math.order).found == 2);
```

Comment:
Same thing here, `std.testing.expectEqual` would grant a nicer error message in case of failure.

Reply:
resolving since we're addressing this in the other pr